qtp_biom/validate.py

```diff
@@ -56,7 +56,10 @@ def validate(qclient, job_id, parameters, out_dir):
         metadata = metadata['data']
 
         qurl = ('/qiita_db/prep_template/%s/' % prep_id)
-        md = qclient.get(qurl)['qiime-map']
+        response = qclient.get(qurl)
+
+        md = join(f'{out_dir}/merged_information_file.txt')
```

Comment:
Is the call for join needed here? You are only inputting one string.

Reply:
left by mistake